templates/client-auth-n.cfg.xml.erb

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <client-auth xmlns="http://<%= scope.lookupvar('::repose::cfg_namespace_host') %>/repose/client-auth/v1.0">
+<%- if @delegating_enabled -%>
```

Comment:
I think you're targeting the wrong file for this? This already exists in this file see lines 8-12